cmd/deployIntegrationArtifact.go

```diff
@@ -77,7 +77,7 @@ func runDeployIntegrationArtifact(config *deployIntegrationArtifactOptions, tele
 
 	defer func() {
 		if deployResp != nil && deployResp.Body != nil {
-			deployResp.Body.Close()
+			defer cpi.CloseResponseBodyIfNecessary(deployResp)
 		}
 	}()
```

Comment:
As the response and the response body are checked in the CloseResponseBodyIfNecessary function, the if here is redundant. Instead the defer can be done with defer cpi.CloseResponseBodyIfNecessary(deployResp)